pdf-rendering-template/bin/render-pdf.js

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
🐘 would be good to apply prettier (our eslint-config would be nice too, but since it's just this js file probably a bit overkill - my comments match what eslint would raise anyway)

Reply:
good point, I'll run our eslint on it

Reply:
done